src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,163 @@
+package duke;
+import java.io.File;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+/**
+ * The class that contains the task list. It has operations to add/delete
+ * tasks in the list.
+ */
+class TaskList {
```

Comment:
It seems like your TaskList is doing the job of Ui returning String objects to be printed/shown to user.